docs/src/TurbulenceConvection.md

```diff
@@ -0,0 +1,37 @@
+# `TurbulenceConvection`
+
+## Grids
+
+```@docs
+CLIMA.TurbulenceConvection.Grids.Grid
```

Comment:
You can add a CurrentModule spec at the top of the markdown so that you don't need to specify all the sub ones, e.g. see https://raw.githubusercontent.com/climate-machine/CLIMA/master/docs/src/AtmosDycore.md